app/scripts/modules/core/src/notification/types/githubstatus/githubstatus.notification.type.module.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
The convention for new files in Deck is to use TypeScript rather than JavaScript.